lib/recurly/pricing/calculations.js

```diff
@@ -69,9 +69,11 @@ Calculations.prototype.subtotal = function () {
   this.price.now.subtotal += this.price.now.plan;
   this.price.next.subtotal += this.price.next.plan;
 
-  this.addons();
-  this.price.now.subtotal += this.price.now.addons;
-  this.price.next.subtotal += this.price.next.addons;
+  if(this.items.plan.addons && this.items.plan.addons.length){
+    this.addons();
+    this.price.now.subtotal += this.price.now.addons;
+    this.price.next.subtotal += this.price.next.addons;
+  }
```

Comment:
This needs to move down into the addons method, since it initially sets the addons values to zero. I recommend bailing after that initial set if there are no addons.